src/WebJobs.Script.WebHost/WebHostResolver.cs

```diff
@@ -112,15 +128,16 @@ internal void EnsureInitialized(WebHostSettings settings)
                     _settingsManager.Reset();
 
                     _activeScriptHostConfig = CreateScriptHostConfiguration(settings);
-                    _activeHostManager = new WebScriptHostManager(_activeScriptHostConfig, _secretManagerFactory, _eventManager,  _settingsManager, settings, _router, _loggerFactoryBuilder);
+                    _activeHostManager = new WebScriptHostManager(_activeScriptHostConfig, _secretManagerFactory, _eventManager, _settingsManager,
+                        settings, _router, loggerProviderFactory: _loggerProviderFactory);
                     //_activeReceiverManager = new WebHookReceiverManager(_activeHostManager.SecretManager);
                     InitializeFileSystem();
 
                     if (_standbyHostManager != null)
                     {
                         // we're starting the one and only one
                         // standby mode specialization
-                        _activeScriptHostConfig.TraceWriter.Info(Resources.HostSpecializationTrace);
+                        _activeHostManager.FallbackLoggerFactory.CreateLogger("Host.Active").LogInformation(Resources.HostSpecializationTrace);
```

Comment:
Can't we have a default logger that we've created already in the ctor or something, so we don't need to do this every time we need to log? So in this code we can just to `_logger.LogInformation("....")`

Reply:
`CreateLogger()` is actually just a dictionary lookup in the logging infrastructure. But I see your point -- I'll create one and hold onto it.